package.json

```diff
@@ -60,13 +75,16 @@
     "express": "^4.16.4",
     "finalhandler": "^1.1.1",
     "glob": "^7.1.3",
+    "husky": "^1.3.1",
```

Comment:
Please remove husky

Reply:
@jsdevel , any particular reason to remove  `husky`?